.github/workflows/ci.yml

```diff
@@ -44,8 +44,7 @@ jobs:
           - 'centos-7'
           - 'oracle-linux'
           - 'amazon-linux'
-          - 'ubuntu-1604'
-          - 'ubuntu-1804'
```

Comment:
I'd keep 18.04 at least. You might also add CentOS 8 but that might want to be a different PR depending how much work that might be.